package.json

```diff
@@ -23,6 +23,7 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.19",
     "@fortawesome/free-solid-svg-icons": "^5.9.0",
     "@fortawesome/react-fontawesome": "^0.1.4",
+    "babel-preset-es2015": "^6.24.1",
```

Comment:
why we need this ?

Reply:
I was attempting to debug an issue but it was not necessary in the end